rest_witchcraft/field_mapping.py

```diff
@@ -57,6 +58,9 @@ def get_url_kwargs(model):
     int: fields.IntegerField,
     str: fields.CharField,
 }
+SERIALIZER_FIELD_MAPPING[six.text_type] = fields.CharField
```

Comment:
`text_type` is always part of `string_types`. any explicit set here?

Reply:
Its not actually in py2:
```python
In [1]: import six

In [2]: six.text_type
Out[2]: unicode

In [3]: six.string_types
Out[3]: (basestring,)

In [4]: 
```